src/compiler/proposals-and-stabilization.md

```diff
@@ -285,14 +285,25 @@ circumstance.
   - Open an RFC describing the motivation for the change and start an FCP to approve, start an FCP.
   - If approved, the change should be accompanied by a blog post announcing the change with a
     notice period of at least one release before the change applies.
```

Comment:
I can't leave a comment on the relevant line because you haven't changed it - but I assume we want to split "Promoting a target" into three individual recommendations for each tier.